product_plan.md

```diff
@@ -0,0 +1,83 @@
+
+## Overview
+
+Our short-term high-level product plan is to build [Spacemesh 0.1](spacemesh01.md), release testnets, follow up with `Spacemesh 0.x` updates and a launch a mainnet for this product according to the project [roadmap](roadmap.md).
+
+Our long-term high-level product plan is to implement, test and roll-out additional capabilities and features to the platform in subsequent incremental releases until the full platform is released as `Spacemesh 1.0` with support to all the main use cases outlined here.
+
+Incremental product releases, starting with `Spacemesh 0.1` will be deployed to testnets first for testing and to the `Spacemesh Mainnet` once they reach production quality status and have been security audited.
+
+## TL;DR
+
+![](https://raw.githubusercontent.com/spacemeshos/product/master/resources/roadmap2019.png)
+
+---
+
+## Spacemesh 0.1
+
+### Release Highlights
+
+The main use case case of `Spacemesh 0.1` is to support [Spacemesh Coin](spacemesh_coin.md) cryptocurrency transactions between any two parties and to award `Spacemesh coins` to people who run full Spacemesh full p2p nodes on their PCs. The Spacemesh full node implements the Spacemesh consensus protocol and other protocols to support this use case.
+
+The main goal of this release is to provide an MVP of a permissionless and trustless Internet money that is secure, decentralized and scalable without using POW or PoStake mechanisms.
+
+### Visual Story
+![Story](https://raw.githubusercontent.com/spacemeshos/product/master/resources/sm_0.1_story.jpg)
+
+### Release main software components
+
+### Use cases
+- [Available here](spacemesh01.md)
+
+### Main Components
+1. [go-spacemesh](https://github.com/spacemeshos/go-spacemesh) - The reference implementation of the Spacemesh full node in go-lang for Windows 10, OS X and Linux.
+
+2. [Spacemesh POET service](https://github.com/spacemeshos/POET) - a public utility used in the Spacemesh proofs of space time protocol
+
+3. [Spacemesh App](https://github.com/spacemeshos/smapp) - A desktop app for Windows 10, OS X and Linux which includes a Wallet, a managed full p2p node and mining.
+
+4. `Spacemesh Gateway` - Foundation deployed nodes providing the `Spacemesh API` for wallets and for the app to check coin balance, and execute transactions.
+
+5. [Spacemesh Tap](tap.md) - Enables users to get `Spacemesh Coins` on testnets for testing purposes.
+
+6. [Spacemesh POST](https://github.com/spacemeshos/post) - Implement the proofs of space protocol used by Spacemesh.
+
+For additional information about these components read this page in the [Spacemesh Testnet guide](https://testnet.spacemesh.io/#/dict)
+
+---
+
+## Spacemesh 0.x
+0.x releases are going to include bug fixes, updates and additional features for testing on the public testnet prior to mainanet launch. The mainnet will be launched once we are comfortable with the quality, security and performance of a 0.x release.
```

Comment:
According to the table, this milestone contains "Tortoise self-healing" - is it co-ordinated with @tal-m ? When will the protocol is expected to be ready? If we want to deliver it by Q1/2020 the protocol must be delivered ASAP

Reply:
This is not one milestone - this is a series of 0.x releases. e.g. 0.2, 0.3, 0.4, ... etc - until we have tested on testnet all implemented features before mainnet. Sol, self-healing and any other feature will be included in a 0.x release - this is high level product plan does not detail in which specific 0.x release any feature will be delivered - it sets the framework and the terminology for these decisions.

Reply:
According to the HL plan, all 0.x series need to be delivered during Q1 2020. If you want to include self-healing in 0.x I need to get the detailed protocol ASAP. When is it due? Where can I find the low-level product plan?